pycbc/types/config.py

```diff
@@ -513,17 +525,29 @@ def check_duplicate_options(self, section1, section2, raise_error=False):
                 "Section %s not present in ConfigParser." % (section2,)
             )
 
+        section_and_default = (section1 == f"{section2}-defaultvalues" or
+                               section2 == f"{section1}-defaultvalues")
```

Comment:
I don't know that we'd ever run this function on two separate sections (ie. `coinc` and `plotcoinc`) as opposed to subsections. Still, I don't object to this fallback, but what to do if `defaultvalues` collide is undetermined because one should not take options from two separate sections.

Reply:
Oh wait, that's an `or` not an `and`